roles/tests/test_permissions.py

```diff
@@ -57,10 +59,14 @@ def test_repo_permissions(self):
             RepoPermission.add_edit_metadata,
             add_edit_metadata
         )
+        self.assertEqual(
+            RepoPermission.manage_repo_users,
+            manage_repo_users
+        )
         self.assertListEqual(
             RepoPermission.administrator_permissions(),
             [view_repo[0], import_course[0],
```

Comment:
nit: This is getting a bit hard to read, maybe switch to:

```python
[
    view_repo[0],
    import_course[0],
    manage_taxonomy[0],
    add_edit_metadata[0]]
     manage_taxonomy[0],
     add_edit_metadata[0],
     manage_repo_users[0],
]
```
it will also have the side effect of make adding a new permission easier in the future